pkg/kubelet/winstats/perfcounters.go

```diff
@@ -0,0 +1,119 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package winstats
+
+import (
+	"errors"
+	"fmt"
+	"github.com/lxn/win"
+	"time"
+	"unsafe"
+)
+
+const (
+	CPUQuery                  = "\\Processor(_Total)\\% Processor Time"
+	MemoryPrivWorkingSetQuery = "\\Process(_Total)\\Working Set - Private"
+	MemoryCommittedBytesQuery = "\\Memory\\Committed Bytes"
+)
+
+// Metric collected from a counter
+type Metric struct {
+	Name  string
+	Value float64
+}
+
+// NewMetric construct a Metric struct
+func NewMetric(name string, value float64) Metric {
+	return Metric{
+		Name:  name,
+		Value: value,
+	}
+}
+
+func (metric Metric) String() string {
+	return fmt.Sprintf(
+		"Name: %s | Value: %s",
+		metric.Name,
+		metric.Value,
+	)
+}
+
+func readPerformanceCounter(counter string, sleepInterval int) (chan Metric, error) {
```

Comment:
I decided to remove the sleepInterval argument and replace it with a constant instead since I was only using this in one place.